client/src/common/helpers/accountConverter.ts

```diff
@@ -0,0 +1,14 @@
+import Keyring from '@polkadot/keyring'
+import { SUBSPACE_ACC_PREFIX } from 'common/constants'
+
+export const accountConverter = (accountId: string): string => {
```

Comment:
maybe something like `formatAddress` is a better name? also, would be nice to add a comment that we're transforming all ss58 formats to a SUBSPACE format because this is how we identify accounts on the backend (for someone less familiar with the context)